jujugui/static/gui/src/app/components/entity-details/test-entity-details.js

```diff
@@ -115,7 +78,7 @@ describe('EntityDetails', function() {
                   'getEntity function not called');
     assert.equal(getEntity.args[0][0], id,
                  'getEntity not called with the entity ID');
-    assert.deepEqual(output,
+    var expected = (
```

Comment:
Assigning the expected output to a variable makes debugging failed tests much easier. Then there's a variable to inspect or log out. We should talk about whether to use this pattern more widely.